pom.xml

```diff
@@ -9,24 +9,22 @@
   
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.bdgenomics.adam</groupId>
-  <artifactId>adam-parent</artifactId>
+  <artifactId>adam-parent_2.10</artifactId>
   <version>0.16.1-SNAPSHOT</version>
   <packaging>pom</packaging>
-  <name>ADAM</name>
+  <name>ADAM_2.10</name>
   <description>A fast, scalable genome analysis system</description>
   <url>http://bdgenomics.org/</url>
 
   <properties>
     <java.version>1.7</java.version>
-    <scala.version>2.10.4</scala.version>
```

Comment:
Very interesting. I am out of my Maven depth, but now somewhat curious about the failure modes if it's at all interesting; no need to go into further detail if not. Seems innocuous enough, as you said.

And, confirming one piece of subtext: this is all happening because you are trying to release different artifacts for `2.10` vs. `2.11` for the first time, right?